src/main/java/ui/components/FilterTextField.java

```diff
@@ -245,21 +254,29 @@ public void setFilterText(String text) {
     }
 
     /**
-     * Sets the 'cancel' callback, which will be called with Esc is pressed and there is
+     * Sets the 'onCancel' callback, which will be called with Esc is pressed and there is
      * no edit to revert.
      */
-    public FilterTextField setOnCancel(Runnable cancel) {
-        this.cancel = cancel;
+    public FilterTextField setOnCancel(Runnable onCancel) {
+        this.onCancel = onCancel;
         return this;
     }
 
     /**
-     * Sets the 'confirm' callback, which will be called when Enter is pressed, or when the
+     * Sets the 'onConfirm' callback, which will be called when Enter is pressed, or when the
      * contents of the field are manually set with {@link #setFilterText}. The callback will
      * be passed the current contents of the field.
      */
-    public FilterTextField setOnConfirm(Function<String, String> confirm) {
-        this.confirm = confirm;
+    public FilterTextField setOnConfirm(Function<String, String> onConfirm) {
+        this.onConfirm = onConfirm;
+        return this;
+    }
+    
+    /**
+     * Sets the 'onHelp' callback, which will be called when F1 is pressed.
```

Comment:
Developers can see the implementation at [`ui.components.KeyboardShortcuts`](https://github.com/HubTurbo/HubTurbo/blob/6e0d10092066fe64fa0415614eba927892fee349/src/main/java/ui/components/KeyboardShortcuts.java#L81-L82).

I see, I will change it to something like:
```java
/**
 * Sets the 'onHelp' callback, which will be called when the user triggers the show_docs keyboard shortcut
 */
```
I suppose line 257: `... called with Esc is pressed ...` 
and line 266: `... called when Enter is pressed ...`
can be left as is since the <kbd>Esc</kbd> and <kbd>Enter</kbd> keybinds are used specially in this class.